modules/yardstick/src/main/java/org/apache/ignite/yardstick/IgniteBenchmarkArguments.java

```diff
@@ -145,6 +152,17 @@
     @Parameter(names = {"-pp", "--printPartitionStats"}, description = "Print partition statistics")
     private boolean printPartStats;
 
+    /** */
+    @Parameter(names = "--allow-operation", variableArity = true, description = "List of cache operations")
+    private List<String> allowOperations = new ArrayList<>();
+
+    /**
+     * @return List of cache operations.
+     */
+    public List<String> allowOperations() {
```

Comment:
Rename to allowedLoadTestOperations and fix the Java doc referring to an identical comment above